doc/topics/grains/index.rst

```diff
@@ -298,3 +298,9 @@ Syncing grains can be done a number of ways, they are automatically synced when
 above) the grains can be manually synced and reloaded by calling the
 :mod:`saltutil.sync_grains <salt.modules.saltutil.sync_grains>` or
 :mod:`saltutil.sync_all <salt.modules.saltutil.sync_all>` functions.
+
+.. note::
+
+    When the :conf_minion:`grains_cache` is set to False, the grains dictionary is built
+    and stored in memory on the minion. Every time the minion restarts or
+    `saltutil.refresh_grains` is run, the grain dictionary is rebuilt from scratch.
```

Comment:
You'll need double back-ticks on references like `saltutil.refresh_grains` to render correctly in restructured text syntax. (Mark-down is single back-ticks for code references, RST is double back-ticks.)

Reply:
okay, going to add that now. so used to MD lately forgot this was in RST. Thank you. change committed.